script/update_kibana_objects.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+import json
+import tempfile
+import shlex
+import shutil
+import subprocess
+import sys
+import os
+
+
+def exec(cmd):
+    """
+    Executes the given command and returns its result as a string
+    """
+    proc = subprocess.Popen(shlex.split(cmd), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    out, err = proc.communicate()
+    code = proc.returncode
+    if err:
+        print(err)
+    if code > 0:
+        sys.exit(code)
+    return out.decode("utf-8")
+
+
+def call(cmd):
+    """
+    Executes the given command while showing progress in stdout / stderr
+    """
+    code = subprocess.call(shlex.split(cmd))
+    if code > 0:
+        sys.exit(code)
+
+
+def main():
+    """
+    Updates the index pattern in kibana according to the current checked out branch in apm-server
+
+    NOTES:
+    - HOME environment variable must be set
+    - apm-server can't be in detached head state
+    - `make update` must have been run previously
+    """
+
+    apm_dir = os.path.join(os.environ['HOME'], "go/src/github.com/elastic/apm-server")
+    os.chdir(apm_dir)
```

Comment:
does this need to change dir (so that config dir is in the same dir?) or can the full path to `apm-server` be used instead?

Reply:
either should work. im curious what is the downside of `chdir`?

Reply:
I think it just makes refactoring simpler.  If the wd is really important for a group I'd suggest using a context manager for the block where the path is important so something like:

```python
# wd doesn't matter to code above this
with working_dir():
  do()
  ...
# wd doesn't matter to code below this
```